agents/mongodb/internal/profiler/parser/parser_test.go

```diff
@@ -30,7 +31,7 @@ import (
 
 func TestNew(t *testing.T) {
 	docsChan := make(chan pm.SystemProfile)
-	a := aggregator.New(time.Now(), "test-id")
+	a := aggregator.New(time.Now(), "test-id", logrus.WithField("component", "aggregator"))
```

Comment:
Please use `make test` locally. It is much faster than Travis CI, especially after the first run when tests for other packages are tested.